Modules/Core/Transform/include/itkCompositeTransform.hxx

```diff
@@ -789,20 +789,16 @@ CompositeTransform<TParametersValueType, NDimensions>::GetFixedParameters() cons
    * it's efficient. */
   this->m_FixedParameters.SetSize(this->GetNumberOfFixedParameters());
 
-  NumberOfParametersType                      offset = NumericTraits<NumberOfParametersType>::ZeroValue();
-  typename TransformQueueType::const_iterator it;
-
-  it = transforms.end();
+  NumberOfParametersType offset = NumericTraits<NumberOfParametersType>::ZeroValue();
 
-  do
+  for (auto it = transforms.rbegin(); it != transforms.rend(); ++it)
   {
-    it--;
     const FixedParametersType & subFixedParameters = (*it)->GetFixedParameters();
     /* use vnl_vector data_block() to get data ptr */
     std::copy_n(
       subFixedParameters.data_block(), subFixedParameters.Size(), &(this->m_FixedParameters.data_block())[offset]);
     offset += subFixedParameters.Size();
-  } while (it != transforms.begin());
+  };
```

Comment:
Extraneous semicolon?

Reply:
Yes, it appears so. Interesting that the style checkers didn't pick this up. As the PR has already been merged into the master branch. To avoid conflict/divergence, I'm going to just merge as is, and fix it in the master branch only.